### PR TITLE
Doc about new way of running tests

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | yes |
| Applies to | all |
| Fixed tickets | - |

Related to https://github.com/symfony/symfony/pull/15575

### CODE REVIEW

File: contributing/code/tests.rst [hunk 1]
```diff
@@ -35,7 +34,7 @@ command:
 
 .. code-block:: bash
 
-    $ phpunit
+    $ php ./phpunit symfony
```
Comment (OskarStark) [inline contributing/code/tests.rst]: is this correct?

my binary is located under ```bin/phpunit```

Comment (nicolas-grekas) [inline contributing/code/tests.rst]: yes it is: we don't want your phpunit, we want symfony's one (see linked PR)

Comment (OskarStark) [inline contributing/code/tests.rst]: :+1:

**Comment by javiereguiluz** [comment]
:+1: let's see if these changes are merged soon.

File: contributing/code/tests.rst [hunk 2]
```diff
@@ -16,14 +16,13 @@ In any case, it's a good practice to run tests locally before submitting a
...
-and Monolog. To do so, :doc:`install Composer </cookbook/composer>` and execute
-the following:
+To run the Symfony test suite, install the external dependencies used during the
+tests, such as Doctrine, Twig and Monolog. To do so,
+:doc:`install Composer </cookbook/composer>` and execute the following:
 
 .. code-block:: bash
 
-    $ composer install
+    $ composer update
```
Comment (xabbuh) [inline contributing/code/tests.rst]: Why do we need `update` here?

Comment (WouterJ) [inline contributing/code/tests.rst]: but have the same result, as the project doesn't have a lock file

Comment (nicolas-grekas) [inline contributing/code/tests.rst]: When one works on a patch and a composer file is changed, only update works, that's why it's more generic

**Comment by xabbuh** [comment]
:+1:

**Comment by xabbuh** [comment]
Thank you @nicolas-grekas.

**Pull request merged**